### DISCUSSION TITLE
PCAP recorder support

### DISCUSSION THREAD

**Discussion by Slepwin** [loxilb-io/loxilb#218] [Ideas]
Due eBPF it's hard to troubleshoot issues and loxicmd/api needed PCAP recorder function to gather network traffic with n-Tuple like on Cilium XDP L4LB.

**Comment by UltraInstinct14** [comment]
This is currently available via : 
1. "loxicmd get ct"
2. Logged CT entries for audit purpose in log file 

Also enhanced version will be available via :
1. Our under-development observability dashboard tool.

We will also review and implement if there are any better ways to present more debugging options to our users.

**Comment by UltraInstinct14** [comment]
@Slepwin  Thanks for your suggestion. Our team found your suggestion to be very important for debugging and added feature support in loxilb. You can find the details [here](https://github.com/loxilb-io/loxilbdocs/blob/main/docs/cmd.md#pcap-recording). We will soon publish a minor release with newly added features. Thanks again for your feedback !!
